controllers/profiles/object_creators.go

```diff
@@ -59,11 +60,15 @@ const (
 	healthStartedInitialDelaySeconds = 10
 )
 
-var defaultDevApplicationProperties = "quarkus.http.port=" + strconv.Itoa(defaultHTTPWorkflowPort) + "\n" +
+var defaultHTTPWorkflowPortIntStr = intstr.FromInt(defaultHTTPWorkflowPortInt)
+
+var defaultDevApplicationProperties = "quarkus.http.port=" + defaultHTTPWorkflowPortStr + "\n" +
 	"quarkus.http.host=0.0.0.0\n" +
 	// We disable the Knative health checks to not block the dev pod to run if Knative objects are not available
 	// See: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/eventing/consume-produce-events-with-knative-eventing.html#ref-knative-eventing-add-on-source-configuration
-	"org.kie.kogito.addons.knative.health-enabled=false\n"
+	"org.kie.kogito.addons.knative.health-enabled=false\n" +
```

Comment:
My only comment here guys @desmax74 @spolti is that according with a chat we had yesterday with @hbelmiro , he's doing a change in the knative eventing addon, and the name of this property will now be

org.kie.kogito.addons.knative.**eventing**.health-enabled 

see that **eventing** was incorporated to the name.

Could you please confirm @hbelmiro 

Other than this, from what I can understand :) in this PR, I think it's good.

Reply:
Actually, the property names are not going to change. They already have that `.eventing.`, but this is not in the docs.
I opened this PR to fix it in the docs: https://github.com/kiegroup/kogito-docs/pull/353

Reply:
@desmax74 can you please update the properties?

Reply:
btw, ins't better to have these flags set on image side with default value if not set?

Reply:
I've fixed on my PR already. We can merge this mine will fup.

Reply:
Updated